src/dependency.rs

```diff
@@ -26,6 +26,9 @@ pub struct Dependency {
     /// If the dependency is renamed, this is the new name for the dependency
     /// as a string.  None if it is not renamed.
     rename: Option<String>,
+
+    /// Features that are exposed by the dependency
+    pub available_features: Vec<String>,
```

Comment:
If we have a setter, do we need to make this `pub`?

Reply:
good point, probably not.

Reply:
correction, it does actually need to be `pub` so that `main.rs` can read them for printing. Or I could add a getter/remove the setter. which would you prefer?

Reply:
Either way.  We already have one `pub` item, so its just continuing the same.  I just personally prefer my structs to be all public or all private :).